core/trino-main/src/main/java/io/trino/sql/planner/planprinter/PlanNodeStatsSummarizer.java

```diff
@@ -216,6 +218,7 @@ else if (windowNodeStats.containsKey(planNodeId)) {
                         new Duration(planNodeBlockedMillis.get(planNodeId), MILLISECONDS),
                         planNodeInputPositions.get(planNodeId),
                         succinctBytes(planNodeInputBytes.get(planNodeId)),
+                        succinctBytes(planNodePhysicalInputDataSize.getOrDefault(planNodeId, 0L)),
```

Comment:
Do we need to add it here ? Physical input data size is only about source nodes AFAIK